lib/handlers/shared.ts

```diff
@@ -231,3 +232,16 @@ export function parseSlippageTolerance(slippageTolerance: string): Percent {
 export function parseDeadline(deadline: string): number {
   return Math.floor(Date.now() / 1000) + parseInt(deadline)
 }
+
+export function parsePortionPercent(portionPercent: string): Percent {
```

Comment:
if you're passing in portionBps here isn't it already denominated in terms of 10_000? I feel like you could just cast it to number.

Reply:
Yea you can still create the Percent instance but can just cast to a number instead of doing Math.floor and dividing?